tests/e2e/specs/settings/settings.test.js

```diff
@@ -177,5 +177,36 @@ test.describe( 'Settings', () => {
 				).toBeVisible();
 			} );
 		} );
+
+		test.describe( 'Service Based Merchants Setup', () => {
+			test.beforeAll( async () => {
+				await settingsPage.mockMCNotConnected();
+				await settingsPage.goto();
+
+				await page.evaluate( () => {
+					window.glaData.serviceBasedMerchant = true;
```

Comment:
@ankitrox Does that work well for you because in my test, `glaData` get overwritten by PHP injecting that variable.

Reply:
This will not be required whn MC account is disconnected. Removed this piece of code.